bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/Button.java

```diff
@@ -53,4 +53,11 @@ default String getIcon() {
     default String getDefault() {
         return null;
     }
+
+    /**
+     * Returns the type of action of the button
+     */
+    default String getButtonType() {
+        return null;
```

Comment:
"button" would be returned by default.

Reply:
@nit23uec Actually changing this to `button` doesn't give the default value of the buttonType, I've instead added a `@Default` to `buttonType` in the sling model. Now the model.json for a normal button will show `"fd:buttonType": "button"` for a normal button. Could you confirm if this is what was intended with this comment?